projects/core/components/primitive-calendar/primitive-calendar.template.html

```diff
@@ -16,8 +16,8 @@
                 <div
                     *ngIf="!itemIsUnavailable(item) || showAdjacent"
                     automation-id="tui-primitive-calendar__cell"
+                    style.color="var(--tui-day-of-week-{{ colIndex + 1 }})"
```

Comment:
Would be nice if it was not `colIndex` but day index so you can color sunday red, no matter if it's the first or the last day.

Reply:
I am not sure if this is convenient. Don't you think if you choose TUI FIRST DAY as Sunday it's more intuitive to colorize it with `--tui-day-of-week-1` ? 🤔

Anyway I pushed fix, take a look (commit has a little math hack due to Sunday index 0 in our `TuiDayOfWeek` enum)

Reply:
In the states they would make Sunday the first day, but still would keep Sunday and Saturday as weekends/colored red. This seems more semantic to me.